intelmq/bots/parsers/eset/parser.py

```diff
@@ -0,0 +1,81 @@
+import json
+import re
+from intelmq.lib.utils import base64_encode, base64_decode
+from intelmq.lib.bot import ParserBot
+
+message_taxonomy_map = {
+    'Host actively distributes high-severity threat in the form of executable code.': 'malware',
```

Comment:
Please use `malware-distribution` instead. `malware` is deprecated and will be removed in version 3.0